elephant/spade.py

```diff
@@ -1001,18 +1062,30 @@ def approximate_stability(concepts, rel_matrix, n_subsets, delta=0, epsilon=0):
 
     Parameters
     ----------
-    concepts:
-
+    concepts: ???
```

Comment:
These two parameters need a description...

Reply:
Addressed in the following commit (858c363)